### DISCUSSION TITLE
Welcome to demoGithub Discussions!

### DISCUSSION THREAD

**Discussion by Anemaygi** [Sintese-Jr/demoGithub#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gl4a-Veloso** [comment]
Eu concordo

**Comment by drimoraes** [comment]
oi mayuuu!!!! muito obrigada por estar aqui com a gente!!

**Comment by melogtm** [comment]
eu concordo

**Reply by renangomescosta** [reply]
Ei discordo

**Reply by melogtm** [reply]
o q eh teu ta guardado, fica tranquilo

**Comment by najusilva** [comment]
oiiiiiii **faz o S**

**Comment by jvpascale** [comment]
salve

**Comment by CaioBritoDev** [comment]
se quiser sim

**Comment by clebenjr** [comment]
Síntese Jr from Brazil!
